.gitignore

```diff
@@ -113,6 +113,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+tmp/
```

Comment:
what is it?

Reply:
allow developers or project use/create some file in tmp folder and prevent pushing them to repo.